src/main/java/de/tum/in/www1/artemis/service/connectors/VersionControlService.java

```diff
@@ -22,6 +21,8 @@
      */
     void addWebHook(URL repositoryUrl, String notificationUrl, String webHookName);
 
+    void addWebHookToCISystem(URL repositoryUrl, String notificationUrl, String webHookName);
```

Comment:
try to merge with addWebHook and make sure to document it properly using JavaDoc

Reply:
Merged this with the other method in the sense that we now have an overloaded method with an optional webhook secret token as a parameter, in case we want to authenticate the hook.